lib/bb/generate_report_request_form.rb

```diff
@@ -7,10 +7,10 @@ class GenerateReportRequestForm < Common::Form
                                 familyhealthhistory seifoodjournal healthcareproviders healthinsurance
                                 seiimmunizations labsandtests medicalevents medications militaryhealthhistory
                                 seimygoalscurrent seimygoalscompleted treatmentfacilities
-                                vitalsandreadings prescriptions medications vaallergies
+                                vitalsandreadings prescriptions vaallergies
                                 vaadmissionsanddischarges futureappointments pastappointments
                                 vademographics vaekg vaimmunizations vachemlabs vaprogressnotes
-                                vapathology vaproblemlist varadiology vahth wellness dodmilitaryservice ).freeze
+                                vapathology vaproblemlist varadiology vahth wellness dodmilitaryservice ).uniq.freeze
```

Comment:
Actually... scratch that - I just tried doing it and it doesn't work. Apparently it's a bug, and you have to do some weird shit with axiom type. The easiest place to do it would of course be in the serializer, but since eligible data classes are also going to be used for validation possibly in the future, I'm thinking bb_parser.rb is the best place to make this happen.

So just change line 44 of bb_parser.rb to the following:

```
{ data_classes: @parsed_json[:data_classes].uniq.compact }
```

And it should do the trick.